micro-deps/micro-deps-spring-config/src/main/groovy/com/ofg/infrastructure/discovery/DependencyResolutionConfiguration.groovy

```diff
@@ -34,6 +35,9 @@ class DependencyResolutionConfiguration {
     @Autowired(required = false)
     private DependencyPresenceOnStartupVerifier dependencyPresenceOnStartupVerifier
 
+    @Autowired(required = false)
```

Comment:
Sounds good to me, thanks for clarification. If no other issues are left, please squash and merge.